components/footer/MenusAndCardMobile.tsx

```diff
@@ -0,0 +1,134 @@
+import { useScript } from "@deco/deco/hooks";
+import Image from "apps/website/components/Image.tsx";
+import { Card } from "site/components/footer/Card.tsx";
+import {
+  Card as CardProps,
+  Column as ColumnProps,
+  Menu as MenuProps,
+} from "site/components/footer/types.ts";
+import Icon from "site/components/ui/Icon.tsx";
+import { useId } from "site/sdk/useId.ts";
+
+interface Props {
+  columns: ColumnProps[];
+  card: CardProps;
+}
+
+export interface ExtendedMenuProps extends MenuProps {
+  id: string;
+}
+
+const handleToogle = (menus: ExtendedMenuProps[]) => {
+  menus.map((menu) => {
+    const menuElement = document.getElementById(menu.id);
+    if (!menuElement) return;
+    const button = menuElement.querySelector("button");
+
+    const itens = menuElement.querySelector("ul");
+    if (!itens || !button) return;
+
+    button.addEventListener("click", () => {
+      const visibleIcon = menuElement.querySelector(".block");
+      const hiddenIcon = menuElement.querySelector(".hidden");
+      if (!visibleIcon || !hiddenIcon) return;
+      const operation = itens.style.height === "0px" ? "open" : "close";
+      visibleIcon.classList.toggle("block");
+      visibleIcon.classList.toggle("hidden");
+      hiddenIcon.classList.toggle("block");
+      hiddenIcon.classList.toggle("hidden");
```

Comment:
Prefira usar o método de acordo, para evitar inconsistências (apesar de não serem teoricamente possíveis), assim a intenção fica mais clara.
```suggestion
      visibleIcon.classList.remove("block");
      visibleIcon.classList.add("hidden");
      
      hiddenIcon.classList.add("block");
      hiddenIcon.classList.remove("hidden");
```